src/app/initial-html.js

```diff
@@ -17,7 +17,7 @@ Hello World
 <!-- /wp:title -->
 
 <!-- wp:heading {"level": 2} -->
-<h2>Welcome to Gutenberg</h2>
+<h2>What is Gutenberg?</h2>
```

Comment:
Is this a change needed for the PR? Asking since it introduces a trivial chance in a test as well and, not sure it's important for some reason.

Reply:
Well only because the post looks weird with "Welcome to Gutenberg" written twice.  Once in the title and the other in this header down in the content.

Reply:
Ah, cool, I see.

Reply:
> Just wanted to check, did you update the JS bundles intentionally for this PR or are they some side effect of merges? If the second one is true I'd say we can just reset those changes. They seem to conflict anyway.

I updated them on purpose so that testing this PR integrated into WPiOS is easier.  I can regenerate them before merging the PR if all looks fine.

> Currently selected block seems to stay selected even if we focus to title

That's interestingly broken.  I'll look into this.